installers/redhat-docker/radiasoft-download.sh

```diff
@@ -60,6 +50,9 @@ Then re-run this command
         install_err "installer does not support os=$install_os_release_id"
     fi
     install_yum_install docker-ce
+    if [[ ! ${redhat_docker_dev:-} ]]; then
```

Comment:
@robnagler "dev" is crummy. I wasn't sure how to pull all the concepts below into one name. Thoughts?

Also, it is a bit annoying that we will have to add this flag when setting up dev machines. My thoughts for doing it this way was that it was safer. Better to have dev machines not do something they should do than rsconf managed machines (prod) do something they shouldn't do.

Reply:
install_channel?

Reply:
I don't think that is granular enough. We could be in a "dev" rsconf setup or a "dev" setup outside of rsconf.

Reply:
What is the case you are setting redhat_docker_dev now?

Reply:
In vm_devbox if someone wants to run with docker then they'll run `redhat_docker_dev=1 radia_run redhat-docker`.

Reply:
Maybe we could know we are inside a VM. It has to have some unique state that we can check. This seems like a feature test, for example, you could look at sudoers for vagrant (sudo -l -U vagrant | grep 'not allowed').

Reply:
Ok, I'll do that. I'm going to close this because I need the work along with work in another branch.